### ISSUE TITLE
How to get the remaining in time instead of points?

### ISSUE THREAD

**Issue by rraallvv** [animir/node-rate-limiter-flexible#8]
I'd like to preset the user a timer with the remaining time, is that possible?

**Comment by animir** [comment]
@rraallvv Could you describe what you want to achieve?

1. There is `duration` option, you can find description here https://github.com/animir/node-rate-limiter-flexible#options
2. If you need to get remaining time before points are reset, there is `msBeforeNext` in both resolve and reject of any action `consume`, `reward`, etc

**Comment by rraallvv** [comment]
@animir Thanks a lot! It did the trick.

By the way, in order to be able to use `node-rate-limiter-flexible` on a back4app application I had to replace the method to get the table name like so:

 ```
 // MongoDB table name for the appplication
  RateLimiterMongo.getDbName = () => "the table name in server settings";
```

**Comment by animir** [comment]
@rraallvv Good catch.
I've published v0.17.2.
It supports `dbName` option for RateLimiterMongo now, but it works only with native MongoDB package, as mongoose connects to exact DB.

**Comment by rraallvv** [comment]
@animir Wow! Thanks again.
